test/models/report_test.rb

```diff
@@ -17,45 +18,39 @@ class ReportTest < ActiveSupport::TestCase
     assert_not @report.editable?(incorrect_user)
   end
 
-  test 'created_onはDateクラスを返す' do
+  test 'created_onはDateクラスかつ日報の作成された日付を返す' do
     @report.created_at = Time.zone.now
-    assert_instance_of Date, @report.created_on
+    assert_equal Date.today, @report.created_on # rubocop:disable Date cop
```

Comment:
```suggestion
    assert_equal Date.current, @report.created_on
```

とすればRubocopに怒られないかも？

todayとcurrentの違いについては以下の記事をご覧ください〜

https://qiita.com/jnchito/items/cae89ee43c30f5d6fa2c

Reply:
ありがとうございます、変更すると確かに怒られなくなりました！
Dateクラスは素のRubyクラスのものであると思い、るりまばかり見ていましたが、
Rails上ではオーバーライドすることにより拡張されているのですね🤔
貼っていただいた伊藤さんのQiita、とても参考になりました。（グローバルサービスって大変そうだと思いました）

RailsAPIのソースコードも確認し、`current`だと（`Time.zone`,`config.time_zone`がセットされていれば）`Time.zone.today`になるため、Rubocopに怒られないということも理解しました。